cirq-core/cirq/transformers/routing/initial_mapper_test.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cirq
+
+from cirq import IdentityInitialMapper
+
+
+def test_identity_initial_mapper():
+    input_map = {cirq.NamedQubit(str(i)): cirq.NamedQubit(str(-i)) for i in range(1, 6)}
+    initial_mapper = IdentityInitialMapper(input_map)
```

Comment:
usually we `import cirq` and do `cirq.Thing` in user code (and therefore test code)